src/HomeKit/HMMutablePresenceEvent.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.HomeKit {
+
+	partial class HMMutablePresenceEvent {
+
+		public HMPresenceType PresenceType {
```

Comment:
Right to avoid a breaking change, fair enough. However that is in the eventuality the generator fix doesn't make it before our release of the Xcode 9 APIs right? We can break before that.